src/packages/index.ts

```diff
@@ -7,4 +7,4 @@ export * from "./ListenRestClient";
 export * from "./ManageRestClient";
 export * from "./ReadRestClient";
 export * from "./SelfHostedRestClient";
-export * from "./SpeakRestClient";
```

Comment:
this file is called a barrel file. it should export all clients that exist in the directory. please export `SpeakRestClient`, `SpeakLiveClient`, and `SpeakClient`

Reply:
For this, I use a tool called auto-barrel which allows you to right-click on a directory and click on "update barrel file", which does it all for you.